applications/machine_learning/Kconfig

```diff
@@ -5,6 +5,7 @@
 #
 
 menu "Machine learning application"
+rsource "Kconfig.ml_app"
```

Comment:
I wonder if it should be moved to src directory (currently we treat Kconfig files as source files). I would not NACK because of that.

Reply:
Could be, no objections.